tests/test-basic-user-only.sh

```diff
@@ -99,3 +100,14 @@ if ! skip_one_without_user_xattrs; then
     $OSTREE fsck -q
     echo "ok hardlink pull from bare-user"
 fi
+
+cd ${test_tmpdir}
+rm repo -rf
+ostree_repo_init repo init --mode=bare-user-only
+rm files -rf && mkdir files
+echo afile > files/afile
+$OSTREE --repo=repo commit ${COMMIT_ARGS} -b perms files
```

Comment:
Shouldn't we be running `ostree fsck` here?

Reply:
Ack, I've added it.
Although this PR is not the fix for https://github.com/ostreedev/ostree/issues/2410.
`ostree fsck` only happens to work here because `${COMMIT_ARGS}` values let this test land on a happy path.